setup

```diff
@@ -181,9 +181,13 @@ pushd "\$SOURCE_DIR"/machines >& /dev/null
 for MACHINE_FILE in \$(ls)
 do
   MACHINE=\${MACHINE_FILE/\.sh/}
-  if echo `hostname` | grep -q "\$MACHINE"; then
+  echo \`hostname\` | grep -q "\$MACHINE" 
+  host_match=\$?
+  echo \$SYSTEM_NAME | grep -q "\$MACHINE"
```

Comment:
We should set SYSTEM_NAME using `hostname` or something. Hopefully that will work with PNNL systems.

Reply:
According to our sys admin, $SYSTEM_NAME should be defined on all our systems - at least I know for sure it's correct on deception & compy. And if $SYSTEM_NAME isn't defined it just returns blank so this logic should still work since `hostname` will cover the other systems?

Reply:
We do something along the lines of this on another project if you'd prefer
```
if [[ ! -v MY_CLUSTER ]]
then
  export MY_CLUSTER=`uname -n | sed -e 's/[0-9]//g' -e 's/\..*//'`
fi

# Correctly identify clusters based on hostname
case $MY_CLUSTER in
  newell*)
    export MY_CLUSTER=newell
    ;;
  dl*|deception|*fat*)
    export MY_CLUSTER=deception
    ;;
  *)
    echo "Cluster $MY_CLUSTER not identified - you'll have to set relevant variables manually."
    ;;
esac
```

Reply:
Ah, got it. I just wasn't paying close enough attention. :-) No, I think what you have is fine. I'll approve again and you can merge.